documentation/ame-guide/modules/ROOT/pages/generate/generate-aasx.adoc

```diff
@@ -1,15 +1,15 @@
 = Generate AASX/XML
 
-To generate an AASX/XML files from the Aspect Model, proceed as follows:
+To generate AASX/XML files from an Aspect Model, proceed as follows:
 
-* Click the *Generate* on xref:getting-started/ui-overview.adoc#menu-generate[menu]. +
-A menu offers one option:
-
-** *AASX/XML* -- continue with this one
+* Click the xref:getting-started/ui-overview.adoc#menu-generate[Generate menu] and select *AASX/XML*.
```

Comment:
Not sure about the "the" here .. If we remove it here, we have to apply that to all following instructions.